src/apps/front-office/shop/pages/ProductDetailsPage/ProductReviews/components/ProductReviewForm/ProductReviewForm.tsx

```diff
@@ -3,9 +3,9 @@ import { Button } from "apps/front-office/design-system/components/ui/button";
 import { Input } from "apps/front-office/design-system/components/ui/input";
 import { Textarea } from "apps/front-office/design-system/components/ui/textarea";
 
-export type ProductReviewFormProps = {
-  // props go here
-};
+// export type ProductReviewFormProps = {
```

Comment:
If not used then it would be better to be removed

Reply:
It has already been committed by someone else